modules/edge-machine-pools-aws-local-zones.adoc

```diff
@@ -23,3 +22,87 @@ The cluster network MTU must be always less than the EC2 MTU to account for the
 
 The network plugin can provide additional features, like IPsec, that also must be decreased the MTU. For additional information, see the documentation.
 ====
+
+{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS Local Zone locations. Because of the type and size limitations of resources like EC2 and EBS on Local Zone resources, the default instance type can vary from the traditional worker pool.
+
+The default Elastic Block Store (EBS) for Local Zone locations is `gp2`, which differs from the regular worker pool. The instance type used for each Local Zone on edge compute pool also might differ from worker pools, depending on the instance offerings on the zone.
+
+The edge compute pool creates new labels that developers can use to deploy applications onto AWS Local Zone nodes. The new labels are:
+
+* `node-role.kubernetes.io/edge=''`
+* `machine.openshift.io/zone-type=local-zone`
+* `machine.openshift.io/zone-group=$ZONE_GROUP_NAME`
+
+////
+By default, the system creates the edge compute pool manifests only if users add AWS Local Zone subnet IDs to the list `platform.aws.subnets`.
+////
+
+By default, the machine sets for the edge compute pool defines the `taint` of `NoSchedule` to prevent regular workloads from spreading on Local Zone instances. Users can only run user workloads if they define tolerations in the pod specification.
```

Comment:
Does `taint` need backticks? 🤔 I searched the openshift-docs repo and I don't see any instances of taint being in backticks. See https://docs.openshift.com/container-platform/4.13/virt/install/virt-specifying-nodes-for-virtualization-components.html#virt-example-node-placement-tolerations-hyperconverged-cr_virt-specifying-nodes-for-virtualization-components

Additionally, is `NoSchedule` a configuration parameter in a the edge compute pool resource? If so I would suggest adding some context, such as "...of the `NoSchedule`  edge compute pool parameter"

Reply:
Correct. I searched https://docs.openshift.com/container-platform/4.13/nodes/scheduling/nodes-scheduler-taints-tolerations.html#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations and `taints` not `taint` is a valid parameter. Dropped backticks from "taint".

Reply:
I provided a links to https://docs.openshift.com/container-platform/4.13/nodes/scheduling/nodes-scheduler-taints-tolerations.html#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations that shows that the `NoSchedule` is a value of the `effect` parameter. @skrthomas and @mtulio , does including a link to "Understanding taints and tolerations" help provide context are taint value of `NoSchedule`?